src/platforms/common/performance/index.mdx

```diff
@@ -35,6 +35,16 @@ redirect_from:
 
 <!-- Include in `notSupported` any SDK which enables tracing automatically -->
 
+<PlatformSection supported={["native"]}>
+
+<Alert level="info" title="Important">
+
+This is currently an experimental feature. Performance monitoring is still a work-in-progress feature, and its API may be unstable. It may also have bugs. Feedback is appreciated, and should be submitted as an issue in the appropriate repository.
+
+</Alert>
```

Comment:
an early adopter/experimental feature is just a regular level note. also seems a bit redundant. i'm not married to the wording edit, feel free to change it back if you feel like the other version is conveying something more.

```suggestion
<Note>

Performance monitoring is still an experimental, work-in-progress feature, and its API may be unstable. It may also have bugs. Feedback is appreciated, and should be submitted as an issue in the appropriate repository.

</Note>
```

Reply:
The edit certainly makes it less wordy so I've decided to add that in. 👍